lib/osf-components/addon/components/support-item/template.hbs

```diff
@@ -0,0 +1,9 @@
+<div local-class='container'>
+    <img data-test-icon-image src='/assets/images/dashboard/{{this.icon}}.svg' alt='{{this.icon}}'>
+    <h2 data-test-support-header>
+        {{this.header}}
+    </h2>
+    <p data-test-support-subheader>
+        {{this.subHeader}}
+    </p>
```

Comment:
I think it would be more semantically correct to this section `description` instead of `subHeader`. Please update everywhere (include translations file).